avalon/tools/loader/widgets.py

```diff
@@ -98,15 +98,21 @@ def __init__(self, enable_grouping=True, parent=None):
         self.view.setModel(self.family_proxy)
         self.view.customContextMenuRequested.connect(self.on_context_menu)
 
-        header = self.view.header()
-        # Enforce the columns to fit the data (purely cosmetic)
-        QtCompat.setSectionResizeMode(
-            header, QtWidgets.QHeaderView.ResizeToContents)
+        view.setColumnWidth(0, 240)  # subset
+        view.setColumnWidth(1, 120)  # family
+        view.setColumnWidth(2, 80)   # version
+        view.setColumnWidth(3, 120)  # time
+        view.setColumnWidth(4, 100)  # author
+        view.setColumnWidth(5, 80)   # frames
+        view.setColumnWidth(6, 60)   # duration
+        view.setColumnWidth(7, 60)   # handles
+        view.setColumnWidth(8, 60)   # step
```

Comment:
> Disabled QTreeView header's ResizeToContents resize mode to improve the performance.

What's the performance difference for this? This doesn't seem like the thing on first sight that would be super but I can imagine it would need to go through all data so I could understand why it is. Just wouldn't have thought this would make a noticeable difference.

Reply:
The improvement might only be noticeable when you have lots of subsets, the GUI may appear a bit laggy when you drag the splitter which will change the layout and make the view to update the header to fit the contents.

 So I thought maybe we could disable it and use default size for each column ?

Actually, I forgot to mention another reason, when version data updated from worker thread, emitting `dataChanged` signal is not enough to trigger the column resize, and the widget update call from the *spinning version delegator* could not trigger that as well.

And one may find the version data like `author`, `date` gets cropped but could not resize the column from header, only drag the splitter or open/close the group item to trigger the layout change could work. :(

Reply:
I have patched the reason that I forgot to mention into the top comment. :)